client/app/components/packages/feis/edit.hbs

```diff
@@ -1,26 +1,43 @@
-<div class="grid-x grid-margin-x">
-    <div class="cell large-8">
+<SaveableForm
+  @model={{@package}}
+  @validators={{array (hash) (hash)}}
+  as |saveableForm|
+>
 
-      <section class="form-section">
-        <h1 class="header-large">
-          Final Environmental Impact Statement (FEIS)
-          <small class="text-weight-normal">
-            {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
-          </small>
-        </h1>
+  <div class="grid-x grid-margin-x">
+      <div class="cell large-8">
```

Comment:
I know this isn't part of your commit, but it looks like the spacing is messed up here again. 

It shouldn't be 4 spaces, it should only be 2 spaces for tabbing. 

Sorry we don't have a better linter to help with this. Can you fix please? Then, you'll need to re-align the tabbing for the rest of the diff below.